src/server/constants/db-constants.js

```diff
@@ -0,0 +1,16 @@
+const constants = {
+  TABLE_ECAS: "ecas",
+  TABLE_VOYAGES: "voyages",
+  TABLE_ORGANIZATIONS: "organizations",
+  TABLE_PORTS: "ports",
+  TABLE_SELECTED_ROUTES: "selected_routes",
+  TABLE_SUGGESTED_ROUTES: "suggested_routes",
+  TABLE_USER_ROLES: "user_roles",
+  TABLE_USERS: "users",
+  TABLE_VESSEL_REPORTS: "vessel_reports",
+  TABLE_VESSELS: "vessels",
+  TABLE_WAYPOINTS: "waypoints",
+  TABLE_FAVORITE_VESSELS: "favorite_vessels"
+};
+
+module.exports = constants;
```

Comment:
Why are you adding a constants file when we agreed not to use it? Further, you are not referencing it anywhere.

Reply:
Oh.... i created it and then i forgot to delete it once we agreed upon not using it.

Reply:
Allright. Make sure to always look at your file diff when you create a PR.